docs/docs/image-filters/runtime-shader.md

```diff
@@ -6,7 +6,13 @@ slug: /image-filters/runtime-shader
 ---
 
 The `RuntimeShader` image filter allows you to write your own [Skia Shader](/docs/shaders/overview) as an image filter.
-This component receive the currently filtered image as a shader uniform (or the implicit source image if no children are provided). 
+This component receives the currently filtered image as a shader uniform (or the implicit source image if no children are provided).
+
+:::info
+
+Because RuntimeShader relies on texture sampling of the Skia drawing, we recommend to apply a technic known as super sampling. [See below](#supersampling).
```

Comment:
```suggestion
Because RuntimeShader relies on texture sampling of the Skia drawing, we recommend to apply a technique known as super sampling. [See below](#supersampling).